articles/azure-fluid-relay/how-tos/rotate-fluid-relay-access-keys.md

```diff
@@ -0,0 +1,82 @@
+---
+description: Learn how to rotate Azure Fluid Relay access keys
+title: Rotate Azure Fluid Relay access keys
+ms.date: 08/13/2024
+ms.service: azure-fluid
+ms.topic: reference
+---
+
+# How to rotate Fluid Relay Server access keys
+This article provides an overview of managing access keys (tenant keys) in Azure Fluid Relay. 
```

Comment:
Should add here a simple explanation of why and when readers should rotate a key.